docs/people-and-groups/managing.md

```diff
@@ -113,6 +113,28 @@ This action will delete any dashboard subscriptions or alerts the person has cre
 
 This action doesn't affect email distribution lists that are managed outside of Metabase.
 
+## Default user accounts
+
+Metabase includes default user accounts to handle various activities. Metabase will exclude these user accounts from the **Admin settings** > **People** tab, so we're documenting the accounts here so you know they're legitimate accounts and not someone trying to spy on your Metabase.
+
+### Anonymous user account
+
+- ID: 0
+- First name: External
+- Last name: User
+- Email: null
+
+Metabase uses this anonymous user account to log anonymous views, for example views of a [public question or dashboard](../questions/sharing/public-links.md). You'll see this account show up in [usage analytics](../usage-and-performance-tools/usage-analytics.md).
+
+### Metabase internal account
+
+- ID: 13371338
+- First name: Internal
+- Last name: Metabase
+- Email: internal@metabase.com
+
+The [Pro](https://www.metabase.com/pro) and [Enterprise](https://www.metabase.com/enterprise) editions include a user account called "Metabase Internal". Metabase uses this account to load the [Metabase analytics](../usage-and-performance-tools/usage-analytics.md) collection into your Metabase. You may see this `internal@metabase.com` account in the logs.
```

Comment:
From v50 we also load sample embedding content https://www.notion.so/metabase/Seed-new-instances-w-an-example-dashboard-18571f9ab18d41f882c39db92de3634d